XMLtoMD/GuidelineXmlToMD/MarkdownOut/MdText.cs

```diff
@@ -0,0 +1,225 @@
+﻿using System;
+using System.Text.RegularExpressions;
+
+namespace MarkdownOut {
+
+    /// <summary>
+    /// Provides static fields and methods to style and format Markdown text.
+    /// </summary>
+     [System.Diagnostics.CodeAnalysis.SuppressMessage("Naming", "INTL0001:Fields _PascalCase", Justification = "From an Old Github Project, not for production")]
+    public static class MdText {
```

Comment:
Lots of string concatenation going on here. Often this can be improved by using a StringBuilder.